Community-Supported/hyper-jupyter-kernel/README.md

```diff
@@ -21,17 +21,19 @@ With the `hyper_kernel` you can integrate HyperAPI's SQL interface directly into
 
 ![Screenshot of jupyter console](media/jupyterlab.png)
 
-First, install jupyter-console and the kernel provided in this repository using
+First, install JupyterLab and the kernel provided in this repository using
 
 ```
-python -m pip install jupyter-console tableauhyperapi git+https://github.com/tableau/hyper-api-samples#subdirectory=Community-Supported/hyper-jupyter-kernel
+python -m pip install jupyterlab tableauhyperapi git+https://github.com/tableau/hyper-api-samples#subdirectory=Community-Supported/hyper-jupyter-kernel
 python -m hyper_kernel.install
 ```
 
+(Depending on your operating system, you might need to replace `python` by `python3`)
```

Comment:
We might want to simply recommend using `python3`, as jupyter lab dropped Python 2 support.

Reply:
yes, users should always uses python3. But some systems (macos for example) ship Python 3 under the name `python` (i.e. without a `3`).
If you have to type `python3` or `python` hence depends on your system

Reply:
changed to use `python3` exclusively